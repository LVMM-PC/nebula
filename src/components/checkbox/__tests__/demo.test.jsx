import demoTest from "../../../../tests/shared/demo-test";

demoTest("checkbox");
