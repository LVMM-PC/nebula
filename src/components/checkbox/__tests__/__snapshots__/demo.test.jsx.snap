// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./docs/.vuepress/components/demo/checkbox/basic.vue correctly 1`] = `<section><label class="nebula-checkbox-wrapper"><span class="nebula-checkbox"><input type="checkbox" class="nebula-checkbox-input" value=""><span class="nebula-checkbox-inner"></span></span><span>Checkbox</span></label></section>`;

exports[`renders ./docs/.vuepress/components/demo/checkbox/check-all.vue correctly 1`] = `
<main>
  <section><label class="nebula-checkbox-wrapper"><span class="nebula-checkbox nebula-checkbox-indeterminate"><input type="checkbox" class="nebula-checkbox-input" value=""><span class="nebula-checkbox-inner"></span></span><span>Check all
    </span></label></section>
  <section>
    <div class="nebula-checkbox-group"><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox nebula-checkbox-checked"><input type="checkbox" class="nebula-checkbox-input" value="Apple"><span class="nebula-checkbox-inner"></span></span><span>Apple</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox"><input type="checkbox" class="nebula-checkbox-input" value="Pear"><span class="nebula-checkbox-inner"></span></span><span>Pear</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox nebula-checkbox-checked"><input type="checkbox" class="nebula-checkbox-input" value="Orange"><span class="nebula-checkbox-inner"></span></span><span>Orange</span></label></div>
  </section>
</main>
`;

exports[`renders ./docs/.vuepress/components/demo/checkbox/controller.vue correctly 1`] = `
<main>
  <section><label class="nebula-checkbox-wrapper"><span class="nebula-checkbox"><input type="checkbox" class="nebula-checkbox-input" value=""><span class="nebula-checkbox-inner"></span></span><span>Unchecked-Enabled
    </span></label></section>
  <section><button type="button" class="nebula-btn nebula-btn-primary nebula-btn-sm"><span>Check</span></button> <button type="button" class="nebula-btn nebula-btn-primary nebula-btn-sm"><span>Disable</span></button></section>
</main>
`;

exports[`renders ./docs/.vuepress/components/demo/checkbox/disabled.vue correctly 1`] = `
<main>
  <section><label class="nebula-checkbox-wrapper"><span class="nebula-checkbox nebula-checkbox-disabled"><input type="checkbox" disabled="disabled" class="nebula-checkbox-input" value=""><span class="nebula-checkbox-inner"></span></span></label></section>
  <section><label class="nebula-checkbox-wrapper"><span class="nebula-checkbox nebula-checkbox-checked nebula-checkbox-disabled"><input type="checkbox" disabled="disabled" class="nebula-checkbox-input" value=""><span class="nebula-checkbox-inner"></span></span></label></section>
</main>
`;

exports[`renders ./docs/.vuepress/components/demo/checkbox/group.vue correctly 1`] = `
<main>
  <section>
    <div class="nebula-checkbox-group"><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox nebula-checkbox-checked"><input type="checkbox" class="nebula-checkbox-input" value="Apple"><span class="nebula-checkbox-inner"></span></span><span>Apple</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox"><input type="checkbox" class="nebula-checkbox-input" value="Pear"><span class="nebula-checkbox-inner"></span></span><span>Pear</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox"><input type="checkbox" class="nebula-checkbox-input" value="Orange"><span class="nebula-checkbox-inner"></span></span><span>Orange</span></label></div>
  </section>
  <section>
    <div class="nebula-checkbox-group"><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox"><input type="checkbox" class="nebula-checkbox-input" value="Apple"><span class="nebula-checkbox-inner"></span></span><span>Apple</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox nebula-checkbox-checked"><input type="checkbox" class="nebula-checkbox-input" value="Pear"><span class="nebula-checkbox-inner"></span></span><span>Pear</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox"><input type="checkbox" class="nebula-checkbox-input" value="Orange"><span class="nebula-checkbox-inner"></span></span><span>Orange</span></label></div>
  </section>
  <section>
    <div class="nebula-checkbox-group"><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox nebula-checkbox-checked nebula-checkbox-disabled"><input type="checkbox" disabled="disabled" class="nebula-checkbox-input" value="Apple"><span class="nebula-checkbox-inner"></span></span><span>Apple</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox nebula-checkbox-disabled"><input type="checkbox" disabled="disabled" class="nebula-checkbox-input" value="Pear"><span class="nebula-checkbox-inner"></span></span><span>Pear</span></label><label class="nebula-checkbox-wrapper nebula-checkbox-group-item"><span class="nebula-checkbox nebula-checkbox-disabled"><input type="checkbox" disabled="disabled" class="nebula-checkbox-input" value="Orange"><span class="nebula-checkbox-inner"></span></span><span>Orange</span></label></div>
  </section>
</main>
`;
